docs/0.4/howto/hosting_a_splinter_node.md

```diff
@@ -0,0 +1,184 @@
+# Hosting a Splinter Node
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Choosing a Hosting Environment
+
+Currently, Splinter is packaged as Docker images hosted on
+[Docker Hub](https://hub.docker.com/u/splintercommunity/). This means
+Splinter can be deployed anywhere a Docker image can run. Most examples will
+focus on Docker Compose, but if you choose to go with a Kubernetes
+deployment, we have an example of our gameroom demo app designed to work well
+with Minikube in the github repository. The [arcade.yaml
+file](https://github.com/Cargill/splinter/blob/master/docker/kubernetes/arcade.yaml)
+can be used as a starting
+point for customizing your environment.
+
+## Hosting a Splinter Node
+
+Detailed information about planning your Splinter deployment can be found in
+[Planning a Splinter Deployment](https://www.splinter.dev/docs/0.4/howto/planning_splinter_deployment.html)
```

Comment:
These links shouldnt need to link to https://www.splinter.dev. They should be in this format for exmample `[Gameroom Technical Walkthrough]({% link docs/0.4/examples/gameroom/walkthrough/index.md %}).`

Reply:
Good catch! Thanks for providing the example so I knew what to do. :P